src/main.rs

```diff
@@ -41,10 +41,21 @@ fn main() -> CliResult {
 fn run_floki_from_args(args: &Cli) -> Result<(), Error> {
     debug!("Got command line arguments: {:?}", &args);
 
-    let config = FlokiConfig::from_file(&args.config_file)?;
-    debug!("Got configuration {:?}", &config);
+    let environ = environment::Environment::gather()?;
+    debug!("Got environment {:?}", &environ);
+
+    let (floki_root, config_file) = match &args.config_file {
+        Some(config_file) => (environ.current_directory.to_path_buf(), config_file.clone()),
```

Comment:
Oh, maybe I'm misunderstanding -C. Does it just cd to the right place, or does it change the underlying mount? I think there is some use case for changing the mount, and I think that's what I thought this does.

Reply:
In fact, where do we interpret -C?

Reply:
(thinking about it - if it's there just to change directory, I'm wondering whether we need it at all. Feel free to convince me).

Reply:
Pretty sure I broke `-C` in the last set of changes - it should be passed into `Environment::gather`, replacing whatever the cwd is. Since everything else uses the `Environment`'s current working directory member, everything else should "just work".

It should be indistinguishable from changing to the supplied directory and invoking `floki` there. 

I'm happy to leave it out for now - better to keep the cli simple until there's a need for this!

Reply:
Ah, I see. Yeah, let's leave it out for now, and add it back in later if we need it.